lib/ex_docker_build/api/docker_remote_api.ex

```diff
@@ -115,4 +115,13 @@ defmodule ExDockerBuild.API.DockerRemoteAPI do
   def create_volume(payload) do
     HTTPoison.post("#{@url}/volumes/create", Poison.encode!(payload), [@json_header])
   end
+
+  @impl Docker
+  def delete_image(image_id) do
+    "#{@url}/images/#{image_id}"
+    |> URI.parse()
+    |> Map.put(:query, URI.encode_query(%{"fromImage" => image_id}))
```

Comment:
it seems query param `fromImage` is not supported only `force` and `prune` https://docs.docker.com/engine/api/v1.37/#operation/ImageDelete can you remove it and add support for both optional query params (force and prune)? it would be something like taking another parameter `opts` defaults to empty list or empty map then try to convert it to query params as `def remove_container(container_id, params \\ %{})`

Reply:
using the prune does not exclude the images, so I kept only the force param.

Reply:
ok, no problem but based on the docs https://docs.docker.com/engine/api/v1.37/#operation/ImageDelete `force` is a boolean query attribute not the `imageId` can you change this?, also, can you make it optional so you can use `DockerBuild.delete_image(id, %{force: true})` or `DockerBuild.delete_image(id) # false as default` (the same as docs)